packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx

```diff
@@ -167,6 +172,10 @@ const BpkInfoBannerInner = ({
     isExpandable && 'bpk-info-banner__header--expandable'
   );
 
+  const childrenContainerClassName = action && isExpandable
```

Comment:
had to do this conditional class to add different spacing. Will need to check with designers as well, since with "action" button implementation was not reviewed.